.github/workflows/foundry.yml

```
@@ -160,8 +193,8 @@ jobs:
       - uses: actions/checkout@v4
         with:
           submodules: recursive
-      
-      # Cache Foundry dependencies
+
+      # Restore Foundry and Forge cache
       - name: Cache Foundry dependencies
```

Capitalize first letters

```suggestion
      - name: Cache Foundry Dependencies
```